src/Microsoft.DocAsCode.MarkdownLite/JsonRenderer.cs

```diff
@@ -0,0 +1,310 @@
+            foreach (var item in token.Tokens)
+            {
+                childContent += renderer.Render(item);
+            }
+            return this.Insert("Blockquote", childContent, Type.Child);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownListBlockToken token, MarkdownBlockContext context)
+        {
+            var type = token.Ordered ? "ol" : "ul";
+            var childContent = StringBuffer.Empty;
+            foreach (var item in token.Tokens)
+            {
+                childContent += Render(renderer, (MarkdownListItemBlockToken) item);
+            }
+            return this.Insert(type, childContent, Type.Child);
+        }
+
+        protected virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownListItemBlockToken token)
+        {
+            var childContent = StringBuffer.Empty;
+            foreach (var item in token.Tokens)
+            {
+                childContent += renderer.Render(item);
+            }
+            return this.Insert("li", childContent, Type.Child);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownHtmlBlockToken token, MarkdownBlockContext context)
+        {
+            var childContent = StringBuffer.Empty;
+            foreach (var item in token.Content.Tokens)
+            {
+                childContent += renderer.Render(item);
+            }
+            return this.Insert("Html", childContent.ToString(), Type.Child);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownParagraphBlockToken token, MarkdownBlockContext context)
+        {
+            var childContent = StringBuffer.Empty;
+            foreach (var item in token.InlineTokens.Tokens)
+            {
+                childContent += renderer.Render(item);
+            }
+            return this.Insert("Paragraph", childContent, Type.Child);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownTextToken token, MarkdownBlockContext context)
+        {
+            return this.Insert("Text:" + token.Content, this.GetSize(), Type.Size);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownTableBlockToken token, MarkdownBlockContext context)
+        {
+            var content = StringBuffer.Empty;
+
+            // header
+            var headerContent = StringBuffer.Empty;
+            for (int i = 0; i < token.Header.Length; i++)
+            {
+                var childContent = StringBuffer.Empty;
+                foreach (var item in token.Header[i].Content.Tokens)
+                {
+                    childContent += renderer.Render(item);
+                }
+                headerContent += this.Insert("headerItem", childContent.ToString(), Type.Child);
+            }
+            content += this.Insert("Header", headerContent.ToString(), Type.Child);
+
+            // Body
```

Comment:
>Body [](start = 15, length = 4)

check the comments, better all starts with uppercase or lowercase to keep consistency?